packages/material-ui-lab/src/SliderStyled/SliderStyled.js

```diff
@@ -94,144 +94,150 @@ const SliderRoot = experimentalStyled(
       marginRight: 20,
     }),
   }),
-  '& .MuiSlider-rail': {
-    display: 'block',
+}));
+
+export const SliderRail = experimentalStyled('span')((props) => ({
```

Comment:
If we were to push further in this direction in the future. I think that we should add the `muiName` to improve the debugging experience.

to turn:

<img width="251" alt="Capture d’écran 2020-10-11 à 12 26 16" src="https://user-images.githubusercontent.com/3165635/95676271-1a209700-0bbd-11eb-9b46-67253f7d63e4.png">

into:

<img width="243" alt="Capture d’écran 2020-10-11 à 12 27 02" src="https://user-images.githubusercontent.com/3165635/95676273-1c82f100-0bbd-11eb-8817-a2ade77f954e.png">

Reply:
Done, the muiName for the `SliderRail` is `MuiSlider-rail` to match the class generated for it.